include/snowflake/client.h

```diff
@@ -403,6 +419,11 @@ typedef struct SF_TIMESTAMP {
     SF_DB_TYPE ts_type;
 } SF_TIMESTAMP;
 
+/**
+ * Initializes an SF_QUERY_RESPONSE_CAPTURE struct.
+ */
+SF_QUERY_RESULT_CAPTURE sf_query_result_capture_init();
```

Comment:
Also why not dynamically allocate this struct and pass it back through a pointer? Just to keep it consistent with the other init functions

Reply:
Sounds good, will do.